cmd/infra/aws/util/util.go

```diff
@@ -31,12 +35,23 @@ func NewConfig(credentialsFile, region string) *aws.Config {
 	return awsConfig
 }
 
-func NewRoute53Config(credentialsFile string) *aws.Config {
-	awsConfig := NewConfig(credentialsFile, "us-east-1")
-	awsConfig.Retryer = client.DefaultRetryer{
+func NewAWSRoute53Config(credentialsFile string, credKey string, credSecretKey string) *aws.Config {
+
+	awsRoute53Config := NewAWSConfig(credentialsFile, credKey, credSecretKey, "us-east-1")
+	awsRoute53Config.Retryer = client.DefaultRetryer{
 		NumMaxRetries:    10,
 		MinRetryDelay:    5 * time.Second,
 		MinThrottleDelay: 10 * time.Second,
 	}
-	return awsConfig
+	return awsRoute53Config
+}
+
+func NewAWSConfig(credentialsFile string, credKey string, credSecretKey string, region string) *aws.Config {
+
+	if credentialsFile == "" {
+		return newConfig(credentials.NewStaticCredentials(credKey, credSecretKey, ""), region)
```

Comment:
why do we need to have 2 funcs with the same name?
Can't we just have a single unit tested function i.e NewAWSConfig?

Reply:
I didn't want to replace NewConfig everywhere its used as I only need access to those four methods. 
I can expand its use to everywhere or change NewAWSConfig to something like `multiCredAWSConfig`?

Reply:
ok, let's just update NewConfig to support the new params and have a single consistent way to create the aws config we use everywhere and behaves deterministically for any given input.
FWIW could also use provider chain https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/#ChainProvider